skxray/fitting/xrf_model.py

```diff
@@ -807,6 +816,98 @@ def setup_element_model(self, elemental_line, default_area=1e5):
                 else:
                     element_mod = gauss_mod
 
+        else:
+            logger.debug('Started setting up pileup peaks for {}'.format(
+                elemental_line))
+
+            element_line1, element_line2 = elemental_line.split('-')
+
+            def get_line_energy(elemental_line):
```

Comment:
Why is this function defined here?  It does not look like you are closing over anything.